include/eve/module/math/regular/impl/geommean.hpp

```diff
@@ -60,6 +59,6 @@ namespace eve::detail
 
     };
     ((that = next(that,args)),...);
-    return if_else(is_even(sizeof...(args)) && is_ltz(sgn),  eve::allbits, sgn*that);
+    return if_else((sizeof...(args) % 2 == 0) && is_ltz(sgn),  eve::allbits, sgn*that);
```

Comment:
What's wrong with is_even btw?

Reply:
I'm investigating it, I fixed it this way so we can check the rest of the behaviour